static/docs/appendix/document-creator/bill-of-lading/ebl-generic-template-config-file.json

```diff
@@ -0,0 +1,100 @@
+{
+  "network": "<Your network>",
```

Comment:
able to rename this json file? understand that there was 1 `generic-template.json` which is the cover letter one, u wanna update both names so they make sense each?

```
generic-template-ebl.json
generic-template-cover-letter.json
```
maybe something ^